portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,39 +31,51 @@
 import java.util.List;
 import com.google.gson.Gson;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that handles comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
   private static class Comment {
-    private String email, comment;
-    public Comment(String email, String comment){
+    private final String email, comment, time;
+    public Comment(String email, String comment, String time){
```

Comment:
It's slightly more typesafe if this is `long time_millis` instead of `string time_millis`, which could be any valid string.

Reply:
sorry, that is actually a formatted date
l'll change the name accordingly